.github/ISSUE_TEMPLATE.md

```diff
@@ -1,3 +1,9 @@
+**PLEASE READ:** Before posting this report, *please* carefully read our **Troubleshooting Guide** at https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ and search the issues page here for your error message and problem description, as the great majority of bugs are either duplicates, or can be fixed on the user side with a few easy steps.
```

Comment:
Please break this at 80 characters or so, to not have it in a single line.

Reply:
I originally had it that way, but when it I tested how it would actually display on Github, it took up a a lot of excessive vertical space, since Github normally soft-wraps at 105 - 115 characters (at least in my browser) but respects the breaks in the file instead of rewrapping it properly and thus takes up ~50% more vertical room. 

I don't like leaving really long lines like this either, but pragmatically, as the intended use is for display on Github, and this file will normally never be seen/directly used (it only has one change in its entire history since the day it was first committed a few years ago, compared to several dozen issues created each day—around a ~5,000:1 difference), it seemed to make more sense to optimize for where it will actually matter 99.96% of the time. 

On the other hand, the narrower columns break nicely (except for the link, which I don't embed since its primarily for use in the raw/text view anyway) and help readability, It's not a big deal but just let me know which you prefer; you can compare them in the following screenshots illustrating the difference:

Editing View:

![image](https://user-images.githubusercontent.com/17051931/34631450-0a69189e-f236-11e7-854a-a0fe1c778587.png)
vs.
![image](https://user-images.githubusercontent.com/17051931/34631479-29100f6e-f236-11e7-8ca1-23df81f088c0.png)


Rendered View:

![image](https://user-images.githubusercontent.com/17051931/34632554-b567bf98-f23b-11e7-93dc-8facf2ba6c75.png)
vs.
![image](https://user-images.githubusercontent.com/17051931/34632588-eda3b51a-f23b-11e7-8a3f-b0ef4e6b273b.png)